### ISSUE TITLE
Profile pages should show profile picture zoomed in, blurred picture in the Megatron

### ISSUE THREAD

**Issue by matt-elliott** [kevdoan/project3#136]

**Comment by ssorpg** [comment]
i think there was a pr that makes the profile image appear in your profile. assuming that's what you meant, we should be able to close this but let me know

**Comment by matt-elliott** [comment]
Yep, I put that through and it looks really nice. Im going to try and style it up with the blur and all that but only if there is time. Keeping it open for now so I don't forget.
